### DISCUSSION TITLE
I get error "WARNING Access denied api:AssumeRole policy:"

### DISCUSSION THREAD

**Discussion by kukrejashyam** [cloud-custodian/cloud-custodian#7742] [AWS]
Hi Team @thisisshi ,
I am able to push the custodian automation on all Prod accounts except for 3 where it throws below error

The existing IAM role has "Assume role" & all the necessary permissions. Moreover, the same automation executes on other environment with same IAM role that custodian assumes to push the automation.

I tried upgrading custodian to latest version but no luck.
Kindly help.
=============================================
(custodian) ubuntu@ip-10-240-48-220:~/custodian/policies/enforce-backup-tags$ c7n-org run -c ../../org/accounts-old.yml -s logs/ -u enforce-backup-tags-us-east-2-prod.yml -r us-east-2 -a 559328016623 --dryrun
2022-09-08 18:51:10,244: c7n_org:WARNING **Access denied api:AssumeRole policy:enforce-backup-tags** account:cdg-groupsoftware-prod region:us-east-2
2022-09-08 18:51:10,255: c7n_org:INFO Policy resource counts Counter()
=============================================

Below is the screen-shot of latest c7-org version

![image](https://user-images.githubusercontent.com/19922253/189213712-7d8d457c-346f-45c5-9a9d-ac9c5c64357f.png)

**Comment by ajkerrigan** [comment]
If your configuration works in most accounts, it's worth looking for differences with those 3 accounts. Some places to look:

- Does the role listed in your c7n-org config exist in those 3 accounts?
- Does the account role's trust policy allow your c7n-org credentials to assume it?
- Are there any Service Control Policies in place that might block your cross-account access?

You may be able to test cross-account permissions outside of c7n-org. Manual AssumeRole calls via the CLI or SDK might be useful, and it's possible that running those `sts:AssumeRole` operations through the [IAM Policy Simulator](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_testing-policies.html) would give some helpful information too.
